Tests/Controller/CRUDControllerTest.php

```diff
@@ -969,15 +969,15 @@ public function testRedirectToWithObject()
 
         $response = $this->protectedTestedMethods['redirectTo']->invoke($this->controller, $object, $this->request);
         $this->assertInstanceOf('Symfony\Component\HttpFoundation\RedirectResponse', $response);
-        $this->assertSame('list', $response->getTargetUrl());
+        $this->assertSame('list?', $response->getTargetUrl());
```

Comment:
This should have `?` if no parameter.

Reply:
This is a problem of the symfony `Router`

Reply:
What do you mean? It currently works on the current branch, we can't merge this. :-/